go/master/client.go

```diff
@@ -101,9 +118,6 @@ func NewClient(opts ...func(*Client) error) (*Client, error) {
 		}
 	}
 	c.ch = make(chan record, c.bufSize)
-	// FIXME: connection is created asyncrosly in monitorMaster go routine,
-	//        ensure the connection is ready for use before calling c.addClient.
-	time.Sleep(time.Second)
```

Comment:
Deleting because I think without this line the program should work (c.conn.Call will block until a connection is established). If deleting this line does not work, I would be happy to revert this change and send another PR to fix it.